client/state/shortcodes/schema.js

```diff
@@ -0,0 +1,22 @@
+export const siteShortcodeSchema = {
+	type: 'object',
+	properties: {
+		body: { type: 'string' },
+		scripts: { type: 'object' },
+		styles: { type: 'object' }
+	}
+};
+
+export const siteShortcodesSchema = {
+	type: 'object',
+	patternProperties: {
+		'^.+$': { ...siteShortcodeSchema }
+	}
+};
+
+export const shortcodesSchema = {
```

Comment:
That remained from the previous version of the reducer. Merged into one schema now.